Counter/app/build.gradle

```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 32
```

Comment:
i think we're skipping 32 (12L Preview) and going straight to 33 , so i would leave the sdkversions as 31 for now.

(32 was 12L Beta (https://developer.android.com/about/versions/12/12L). )

Reply:
Thank you Caren for the review. sdk versions reverted to 31.